archivy/templates/dataobjs/show.html

```diff
@@ -107,6 +114,46 @@ <h2 id="post-title">{{ dataobj['title'] }}</h2>
           oldContent = document.getElementById("content"), editorDiv = document.querySelector(".EasyMDEContainer");
           editorDiv.classList.add("hidden");
           document.getElementById("post-title").scrollIntoView(true);
+
+          /* Screenreader mode checkbox */
+          function toggleScreenReaderMode(cbox, editor)
+          {
+            return function ()
+            {
+              if (cbox.checked) {
+                editor.toTextArea();
+                editor = null;
+              } else {
+                location.reload();
```

Comment:
Could you explain why the page is reloaded here? I understand for when the button is checked, but don't understand the other side.

Reply:
Ok so, I needed a fast way to revert to the previous state (altough I don't think any screenreader user would deactivate the checkbox , maybe there is some circunstance under which the user might want to revert to default state), and I think it is a more direct, effective and clean (in terms of code readability) way of doing it than reinstantiating easyMDE :)